renderdoc/os/posix/apple/apple_stringio.cpp

```diff
@@ -326,6 +326,13 @@ void WriteOutput(int channel, const char *str)
     fprintf(stdout, "%s", str);
   else if(channel == OSUtility::Output_StdErr)
     fprintf(stderr, "%s", str);
+  else if(channel == OSUtility::Output_DebugMon)
+  {
+    if(OSUtility::DebuggerPresent())
+    {
+      fprintf(stdout, "%s", str);
+    }
+  }
```

Comment:
This will need to be gated so that it only outputs during replay, if there's no specific debugger channel. We can't use stdout/stderr on capture as they may be processed by the application in some way (or a child process).

As a style note, all elses in an if chain should either all have braces or none of them - so this could be rewritten as `else if(channel == OSUtility::Output_DebugMon && OSUtility::DebuggerPresent())` or else add braces to the previous elses.